modules/persistent-storage-csi-vol-populator-procedure.adoc

```diff
@@ -0,0 +1,139 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-vol-populators.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="persistent-storage-csi-vol-populator-procedure_{context}"]
+= Creating pre-populated volumes using volume populators
+
+The following procedure explains how to use a custom data source, using the example 'hello, world' Custom Resource Definition (CRD) created previously, to create a pre-populated persistent volume claim (PVC)."
```

Comment:
For clarity sake, we should mention what it does i.e create a file called example.txt that contains the string "Hello, world!" in the root directory of the volume.

Reply:
Absolutely, there is no real "custom data source" in this case, it's just a populator creating a file with some content passed in as a parameter (in the dataSourceRef field). In real use case there would be some real source, like a bucket for example.

Reply:
Done.